queen-domain/src/test/java/fr/insee/queen/domain/surveyunit/service/SurveyUnitApiServiceTest.java

```diff
@@ -33,31 +31,19 @@ void init() {
         surveyUnitFakeDao = new SurveyUnitFakeDao();
         stateDataFakeService = new StateDataFakeService();
         campaignExistenceFakeService = new CampaignExistenceFakeService();
-        questionnaireModelExistenceFakeService = new QuestionnaireModelExistenceFakeService();
         surveyUnitApiService = new SurveyUnitApiService(surveyUnitFakeDao, stateDataFakeService,
-                campaignExistenceFakeService, questionnaireModelExistenceFakeService, cacheManager);
+                campaignExistenceFakeService, cacheManager);
     }
 
     @Test
-    @DisplayName("On creating survey unit, check campaign existence")
-    void testCreate01() throws StateDataInvalidDateException {
-        StateData stateData = new StateData(StateDataType.VALIDATED, 800000L, "5");
-        SurveyUnit surveyUnit = new SurveyUnit("11", "campaign-id", "questionnaire-id", "[]", "{}", "{}", stateData);
-        surveyUnitFakeDao.setSurveyUnitExist(false);
-        surveyUnitApiService.createSurveyUnit(surveyUnit);
-
-        assertThat(campaignExistenceFakeService.isCheckCampaignExist()).isTrue();
-    }
-
-    @Test
-    @DisplayName("On creating survey unit, check questionnaire existence")
```

Comment:
is it necessary to remove this test?
isCheckCampaignLinkedToQuestionnaire sould be tested for missing campaign too, not just for wrong link between campaign and questionnaireModel